config/katello.migrations/171026094113-rename-cp-oauth.rb

```diff
@@ -0,0 +1,2 @@
+answers['katello'].delete('oauth_key') if answers['katello']['oauth_key']
+answers['katello'].delete('oauth_secret') if answers['katello']['oauth_secret']
```

Comment:
This is an edge case, but this should perhaps be `.key?('oauth_secret')` checks, rather than checks on the values. If someone, one of these was set to a falsey value, this would fail.